release/pkg/aws/ecr/ecr.go

```diff
@@ -119,6 +119,18 @@ func FilterECRRepoByTagPrefix(ecrClient *ecr.ECR, repoName, prefix string, hasTa
 	} else {
 		filteredImageDetails = imageTagFilterWithout(imageDetails, prefix)
 	}
+
+	// Filter out any tags that don't match our prefix for doubletagged scenarios
+	for _, detail := range filteredImageDetails {
+		if len(detail.ImageTags) > 0 {
```

Comment:
Doesn't range work on an empty slice?

Reply:
Are you asking if this would fail on an empty slice?

Reply:
O I see what your'e saying it would just do 0 iterations and go to the next step.